cob_base_controller_utils/launch/halt_detector.launch

```diff
@@ -7,6 +7,7 @@
   <arg name="component_name" default="base" />
 
   <group ns="$(arg component_name)">
+    <rosparam param="halt_detector/diagnostic_recovery_trigger">['ff20#0']</rosparam>
```

Comment:
make this an entry (list of strings) in a robot specific yaml file in mojin_hardware_config